### PR TITLE
[GH-541] Add map collisionables version 3

### LINKED ISSUE #541: [REFACTOR] Colliders position according to concept

### PR DESCRIPTION
**Pull request by agustinesco**
Closes #541

Added new map obstacles according to new map design and adjust spawn positions

### CODE REVIEW

File: apps/arena/priv/config.json [hunk 1]
```diff
@@ -70,6 +70,772 @@
...
+            "y": 1485.0
+          },
+          {
+            "x": -425.0,
+            "y": 1416.0
+          }
+        ]
+      },
+      {
+        "collider": "carapace 1",
```
Comment (AminArria) [inline apps/arena/priv/config.json:143]: What is this field?

**Review by AminArria — APPROVED** [review]

File: apps/arena/priv/config.json (hunk 1 above)
Comment (agustinesco) [reply apps/arena/priv/config.json:143]: this a field that we'll use to identify the rocks in the map, it's really hard to identify where a polygon belongs without this field

**Review by manucamejo — DISMISSED** [review]

File: apps/arena/priv/config.json (hunk 1 above)
Comment (Nico-Sanchez) [reply apps/arena/priv/config.json:143]: Can't it be the "name" instead of "collider"? I think that's a word of unity business logic, not ours

Comment (agustinesco) [reply apps/arena/priv/config.json:143]: sure! yeah the name isn't important, we just want a way to recognize which polygon correspond with which collider

**Review by Nico-Sanchez — APPROVED** [review]

**Review by manucamejo — APPROVED** [review]

**Pull request merged**